src/unix/linux/android/audio.c

```diff
@@ -126,7 +130,13 @@ static void audio_start(MTY_Audio *ctx)
 		AAudioStreamBuilder_setDeviceId(ctx->builder, AAUDIO_UNSPECIFIED);
 		AAudioStreamBuilder_setSampleRate(ctx->builder, ctx->sample_rate);
 		AAudioStreamBuilder_setChannelCount(ctx->builder, ctx->channels);
-		AAudioStreamBuilder_setFormat(ctx->builder, AAUDIO_FORMAT_PCM_I16);
+		/* XXX ATTN Ronald: Requires bumping up android platform from 28 to 32.
+			If OK, let me know and I can uncomment this line. Otherwise, I'll get rid of it
+		AAudioStreamBuilder_setChannelMask(ctx->builder, ctx->channels_mask ?
+			(aaudio_channel_mask_t) ctx->channels_mask : AAUDIO_UNSPECIFIED);
+		*/
```

Comment:
@RonaldH-Parsec Did you check this out? I have stats, but here's not the place to share them. Android API 32 is Android 12